Update builds on ubuntu-16.04 to ubuntu-latest as ubuntu-16.04 is depreciated

ubuntu-16.04 is depreciated we need to update to ubuntu-latest( 18).

First try got error for all jdk versions:

`configure: error: C compiler cannot create executables`, which probably need to update the dependencies base on ubuntu20. Need to confirm that how openj9 set up the dependencies for ubuntu20.

https://github.com/sophia-guo/runaqaTest/runs/2669520807?check_suite_focus=true

config.log: 

```
configure:2956: checking whether the C compiler works
configure:2978: gcc    conftest.c  >&5
/usr/bin/ld: cannot find crt1.o: No such file or directory
/usr/bin/ld: cannot find crti.o: No such file or directory
collect2: error: ld returned 1 exit status
configure:2982: $? = 1
c
```

https://stackoverflow.com/questions/6329887/compiling-problems-cannot-find-crt1-o

Will try `export LIBRARY_PATH=/usr/lib/x86_64-linux-gnu:$LIBRARY_PATH`

--- a/src/builder.ts
+++ b/src/builder.ts
@@ -153,6 +153,7 @@ async function installLinuxDepends(version: string): Promise<void> {
   await exec.exec(`sudo ln -s /usr/include/x86_64-linux-gnu/* /usr/local/include`)
   await exec.exec(`sudo ln -sf /usr/local/bin/g++-7.3 /usr/bin/g++`)
   await exec.exec(`sudo ln -sf /usr/local/bin/gcc-7.3 /usr/bin/gcc`)
+  process.env.LIBRARY_PATH=`/usr/lib/x86_64-linux-gnu:${process.env.LIBRARY_PATH}`
 
 }
 

